crypto/aes/aes-file-encrypt.c

```diff
@@ -140,7 +140,7 @@ int AesEncrypt(Aes* aes, byte* key, int size, FILE* inFile, FILE* outFile)
 }
 
 /*
- * Decryptsr a file using AES 
+ * Decryptsr a file using AES
```

Comment:
Since this has so many typo fixes might as well get this one too, "Decrypts" vs "Decryptsr"